tests/test-samba-container.sh

```diff
@@ -1,43 +1,71 @@
 #!/bin/bash
 
 
+
+echo "determining container command."
 if [ -z "${CONTAINER_CMD}" ]; then
-	CONTAINER_CMD=$(command -v docker || echo "")
+	CONTAINER_CMD=$(command -v docker || echo )
 fi
 if [ -z "${CONTAINER_CMD}" ]; then
-	CONTAINER_CMD=$(command -v podman || echo "")
+	CONTAINER_CMD=$(command -v podman || echo )
 fi
-
+if [ -z "${CONTAINER_CMD}" ]; then
+echo "Error determining container command."
+exit 1
+fi
+echo "container command: '${CONTAINER_CMD}'."
+echo "creating temporary directory."
 TMPDIR="$(mktemp -d)"
 rc=$?
 
 if [ $rc -ne 0 ]; then
-	echo "Error creating temporary directory"
-else
+	echo "Error creating temporary directory."
+	exit 1
+fi
+	echo "temporary directory: '${TMPDIR}'"
+	echo "starting Samba container."
 	CONTAINER_ID="$(${CONTAINER_CMD} run --network=none --name samba \
 		--volume="${TMPDIR}":/share:Z --rm  -d "${LOCAL_TAG}")"
 	rc=$?
 
 	if [ $rc -ne 0 ]; then
 		echo "Error running samba container"
-	else
+		exit 1
+	fi
 		echo "Container started, ID: '${CONTAINER_ID}'"
 
 		# give samba a second to come up
 		sleep 1
 
+		echo "Listing samba shares"
```

Comment:
Wrong indentation: should be one-tab less (including the 3 lines above and the following block).

Reply:
fixed, thanks!